mlir/lib/Dialect/SparseTensor/Transforms/Utils/SparseTensorLevel.cpp

```diff
@@ -62,26 +65,35 @@ class SparseLevel : public SparseTensorLevel {
 
 class DenseLevel : public SparseTensorLevel {
 public:
-  DenseLevel(unsigned tid, Level lvl, Value lvlSize, bool encoded)
-      : SparseTensorLevel(tid, lvl, LevelFormat::Dense, lvlSize),
-        encoded(encoded) {}
+  DenseLevel(unsigned tid, Level lvl, Value lvlSize)
+      : SparseTensorLevel(tid, lvl, LevelFormat::Dense, lvlSize) {}
 
-  Value peekCrdAt(OpBuilder &, Location, Value pos) const override {
-    return pos;
+  Value peekCrdAt(OpBuilder &, Location, ValueRange, Value) const override {
+    llvm_unreachable("locate dense level instead");
```

Comment:
I understand that the error message for BatchLevel is `locate dense level instead`. But why is it the same message in DenseLevel as well?